app/models/user.rb

```diff
@@ -439,7 +439,10 @@ def update_last_wrangling_activity
     return unless is_tag_wrangler?
 
     last_activity = LastWranglingActivity.find_or_create_by(user: self)
+    last_activity.notified_inactive_wrangler = false
+    last_activity.notified_inactive_supervisors = false
     last_activity.touch
```

Comment:
Is `touch` still necessary here?

Reply:
Yes, without it if the two attributes are already false Rails will skip setting saving the record, including the new `updated_at`. This solution feels inelegant though, because it results in two database queries (the touch and the save!), but I couldn't find anything better.